src/Navigation/Navigation.Xamarin.Forms/Models/Configuration.cs

```diff
@@ -1,31 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-
-using XF = Xamarin.Forms;
+﻿using XF = Xamarin.Forms;
 
 namespace CodeMonkeys.Navigation.Xamarin.Forms
 {
     /// <summary>
     /// Settings for the ViewModelNavigationService
     /// </summary>
-    public class Configuration
+    public class Configuration :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the name of type 'Configuration' to be different from an existing framework namespace.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)